Services/PDFGeneration/classes/renderer/wkhtmltopdf/class.ilWkhtmlToPdfConfig.php

```diff
@@ -0,0 +1,1223 @@
+<?php
+require_once './Services/PDFGeneration/classes/class.ilPDFGenerationConstants.php';
```

Comment:
Do we really need the `require_once` directives?

Reply:
No, they are still from the plugin, they will be removed after the final cleanup.